doc/content/source/kernels/BodyForceLM.md

```diff
@@ -1,20 +1,14 @@
 # BodyForceLM
 
-!alert construction title=Undocumented Class
-The BodyForceLM has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /Kernels/BodyForceLM
 
 ## Overview
 
-!! Replace these lines with information regarding the BodyForceLM object.
-
-## Example Input File Syntax
-
-!! Describe and include an example of how to use the BodyForceLM object.
+This object is equivalent to [BodyForce.md] except it adds its residual both to
+the primal equation on which the body force is being applied and to a Lagrange
+Multiplier (LM) equation when an LM is present for enforcing non-negative
+concentrations and we are trying to remove the saddle point due to no
+on-diagonal dependence in the LM equation. See [LMKernel.md] for more details.
```

Comment:
```suggestion
This object is equivalent to [BodyForce.md] except it adds its residual both to
the primal equation on which the body force is being applied and to a Lagrange
Multiplier (LM) equation when an LM is present. We use an LM for enforcing non-negative
concentrations and when we are trying to remove the saddle point due to no
on-diagonal dependence in the LM equation. See [LMKernel.md] for more details.
```

This was a bit too long and run-on like for me, so I suggest breaking it up a little.